src/libraries/System.Text.RegularExpressions/tests/UnitTests/RegexReductionTests.cs

```diff
@@ -470,6 +470,10 @@ public void PatternsReduceIdentically(string actual, string expected)
         [InlineData("(?:ab??){2}", "(?:a(?>b??)){2}")]
         [InlineData("(?:ab??){2, 3}", "(?:a(?>b??)){2, 3}")]
         [InlineData("ab??(b)", "a(?>b??)(b)")]
+        [InlineData(@"\w+\b\w+", @"(?>\w+)\b\w")]
```

Comment:
I'm confused, isn't this reduction what the bug fix is supposed to block? 

edit, nuts, again I'm confused by Github showing the method name above, but this applies to PatternsReduceDifferently